### PR TITLE
Backup and audit logs config for on-prem

### PR DESCRIPTION
**Pull request by prashanth26**
This PR adds documentation to enable backup-restore and user events audit logs for on-prem customers.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,9 +15,9 @@
     "typecheck": "tsc"
   },
   "dependencies": {
-    "@docusaurus/core": "^2.3.1",
+    "@docusaurus/core": "^2.4.1",
```
Comment (prashanth26) [inline package.json:18]: Please take a look at if the dependencies changes are okay. I am not a newbie with `nodejs`. Please suggest changes if required. Thanks.

Comment (slyons) [reply package.json:18]: Let's not update the versions in document change branches.

**Comment by slyons** [comment]
@prashanth26 Please check locally, I added some tables to the documents you wrote.

**Comment by prashanth26** [comment]
> @prashanth26 Please check locally, I added some tables to the documents you wrote.

Sure Scott Thanks. Will check it out and do the needful.

**Comment by prashanth26** [comment]
Hey @slyons ,

As recommended by you I have gotten rid of all the version changes. Just kept mine and your doc changes. Please take a look if it looks good now? 

Thanks.

**Comment by prashanth26** [comment]
pinging @slyons . PTAL. Thanks.

**Review by slyons — APPROVED** [review]
LGTM

**Pull request merged**